src/utils/customMessages.js

```diff
@@ -51,7 +51,9 @@ const messages = {
   usernameExistOrEmpty: 'The username you entered already exists or is empty, if you want to continue update your profile, change the new username or unselect it to remain with the old one',
   passwordChangeSuccess: 'Your password has been changed, login with new password',
   changeEmailNotAllowed: 'You cannot change your email, unselect it if you want to continue updating your profile, otherwise you will not be able to continue',
-  tokenInvalid: 'Authentication failed. Token is invalid'
+  tokenInvalid: 'Authentication failed. Token is invalid',
+  placesRetrieved: 'Successfully retrieved destinations and their visit times',
+  noTripYet: 'There is no trip has done yet'
```

Comment:
Maybe you should change this to `noPlacesRetrieved: 'Data not available for now, please try again later'`

Reply:
I changed it and made it even better for description